crates/store/src/db/sql/mod.rs

```diff
@@ -438,6 +470,15 @@ pub fn upsert_accounts(
     let mut count = 0;
     for update in accounts {
         let account_id = update.account_id();
+        // Extract the 30-bit prefix to provide easy look ups for NTB
+        // TODO: this is hacky:
+        // Do not store prefix for accounts that are not network
+        let account_id_prefix = if account_id.is_network() {
+            NetworkAccountPrefix::try_from(account_id)?.inner()
+        } else {
+            0
+        };
```

Comment:
Related to one of the previous comments: if we make `id_prefix` track only prefixes of network accounts, then it could be a nullable filed and we could set it to `NULL` for non-network accounts.

Reply:
Yes, I made it nullable and created an index for it, should make for fast lookups and less bloated row size since most accounts will not be network accounts. A better alternative would be for it to have its own table but for now this should do OK.